### PR TITLE
Better CMake support

### PR DESCRIPTION
**Pull request by chrhaase**
This is an answer to [this warning](https://github.com/sudara/melatonin_blur/blob/4467c76f99b6098c42edf92d905a45f15fab0657/CMakeLists.txt#L83).
For my needs it's enough to just remove the warning and add the module. I'm not sure what else could be needed?
Maybe a check for JUCE like in [`melatonin_inspector`](https://github.com/sudara/melatonin_inspector/blob/d048df2b227df1ccbd940fc2480f89690dc2d5d5/CMakeLists.txt#L31) would be nice?

### CODE REVIEW

**Comment by sudara** [comment]
Thanks for this. Yeah, this was just like... "I can't deal with this, drawing a line here for now!" — a check for JUCE would be nice!

**Comment by chrhaase** [comment]
Makes sense! Added the check now :)

**Comment by sudara** [comment]
Thanks for this! 

That macos-latest is being intermittent on main as well, so I'll merge anyway. I'm sort of confused why the runs are being cancelled, but I might look into removing the benchmark stuff from the CI runs, they take much longer.

**Comment by sudara** [comment]
> The job running on runner GitHub Actions 3 has exceeded the maximum execution time of 10 minutes.

Ok, wow.... this is new....

File: CMakeLists.txt [hunk 1]
```diff
@@ -80,5 +80,5 @@ if (MelatoninBlur_IS_TOP_LEVEL)
         target_compile_options(Benchmarks PUBLIC $<$<CONFIG:RelWithDebInfo>:-Ofast>)
     endif ()
 else ()
-    message(WARNING "This CMake config is just for CI tests.\nSubmit an Issue / PR if you want more CMake support: https://github.com/sudara/melatonin_blur ")
+    juce_add_module("${CMAKE_CURRENT_SOURCE_DIR}")
```
Comment (7sharp9) [inline CMakeLists.txt:87]: When I tried this today, this is the line that fails on macOS.  I get a:
>Could not locate module header for module

**Pull request merged**